### ISSUE TITLE
NTR: Glycolic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8624]
New term request Glycolic acid high in urine (segal_040303233420): Glycolate  UMLS: 1536029

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000431
- label: Elevated urinary glycolic acid level
- parents: [Elevated urinary monocarboxylic acid level (HP:0033303)](https://hpo.jax.org/app/browse/term/HP:0033303)
- pmid: PMID:4701948
- def: The amount of glycolic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8624
- synonyms:
